src/IssueViz/Models/AnalysisIssueVisualization.cs

```diff
@@ -94,17 +94,27 @@ protected virtual void NotifyPropertyChanged([CallerMemberName] string propertyN
     public static class AnalysisIssueVisualizationExtensions
     {
         /// <summary>
-        /// Returns primary and secondary locations of IssueVisualization
+        /// Returns primary and all secondary locations of the given <see cref="issueVisualization"/>
         /// </summary>
         public static IEnumerable<IAnalysisIssueLocationVisualization> GetAllLocations(this IAnalysisIssueVisualization issueVisualization)
         {
             var primaryLocation = issueVisualization;
-            var secondaryLocations = issueVisualization.Flows.SelectMany(x => x.Locations);
+            var secondaryLocations = issueVisualization.GetSecondaryLocations();
 
             var allLocations = new List<IAnalysisIssueLocationVisualization> {primaryLocation};
             allLocations.AddRange(secondaryLocations);
 
             return allLocations;
         }
+
+        /// <summary>
+        /// Returns all secondary locations of the given <see cref="issueVisualization"/>
+        /// </summary>
+        public static IEnumerable<IAnalysisIssueLocationVisualization> GetSecondaryLocations(this IAnalysisIssueVisualization issueVisualization)
```

Comment:
New extension method, just for convenience. It's a repeating check that we do in additional places, e.g. the lightbulb and tagging, that we could in the future replace with this.